### DISCUSSION TITLE
Possible hardware, Axelera AI

### DISCUSSION THREAD

**Discussion by jonsmirl** [toverainc/willow#95] [General]
https://www.axelera.ai/early-access-program/
120TOPS M.2 projected retail $150

This place will send you a free development unit if you can convince them you know what you are doing.
Looks like a cost effective home solution.

**Comment by kristiankielhofner** [comment]
Thanks for the reminder! I've seen them before but (somehow) didn't connect them in my mind to Willow + Willow Inference Server.

I'll look into it!
